.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report for cuSpatial
+title: "[BUG]: "
+labels: ["bug", "? - Needs Triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuSpatial and fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of cuSpatial are you running?
+      placeholder: "example: 22.10"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installation-method
+    attributes:
+      label: Which installation method(s) does this occur on?
+      multiple: true
+      options:
+        - Docker
+        - Conda
+        - Source
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug.
+      description: Also tell us, what did you expect to happen?
+      placeholder: XYZ occured, I expected QRS results
+    validations:
+      required: true
+
+  - type: textarea
+    id: mvr
+    attributes:
+      label: Minimum reproducible example
+      description: Please supply a [minimum reproducible code example](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) here
+      render: shell
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please paste relevant error and log output here
+      render: shell
+
+  - type: textarea
+    id: env-details
+    attributes:
+      label: Environment details
+      description: Please provide any relevant environment details
+      placeholder: |
+       Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
+       If method of install is Docker, provide `docker pull` & `docker run` commands used
+      render: shell
+
+  - type: textarea
+    id: misc
+    attributes:
+      label: Other/Misc.
+      description: Please enter any other helpful information here.
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+      options:
+        - label: I agree to follow cuSpatial's Code of Conduct
+          required: true
+        - label: I have searched the [open bugs](https://github.com/rapidsai/cuspatial/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report
+          required: true
```

Comment:
This is going to get very irritating for internal team members. I think it's sufficient to say "by submitting, you agree". And I don't want to have to check either of these boxes every time I submit...

Reply:
I think it's nice to have for external contributors, but totally admit that those aren't the majority today and after some trial/error we can't have it only require for ECs. Will reword and remove the checkboxes.

Reply:
Put a sample of how we could handle this in the new doc request template. If it looks good, I'll replace all the others.